src/services/codefixes/fixCannotFindModule.ts

```diff
@@ -23,9 +27,12 @@ namespace ts.codefix {
         return { type: "install package", file: fileName, packageName };
     }
 
-    function getTypesPackageNameToInstall(host: LanguageServiceHost, sourceFile: SourceFile, pos: number): string | undefined {
+    function getTypesPackageNameToInstall(host: LanguageServiceHost, sourceFile: SourceFile, pos: number, diagCode: number): string | undefined {
         const moduleName = cast(getTokenAtPosition(sourceFile, pos, /*includeJsDocComment*/ false), isStringLiteral).text;
         const { packageName } = getPackageName(moduleName);
-        return host.isKnownTypesPackageName(packageName) ? getTypesPackageName(packageName) : undefined;

```

Comment:
Given that we arent using ```getTypesPackageName``` remove it ?

Reply:
ok i see you moved that later, but dont cant you just make it "@types/node" here and getTypesPackageName(packageName) in else branch instead ?

Reply:
Done